tests/framework/extensions/provisioning/verify.go

```diff
@@ -399,20 +400,28 @@ func VerifySnapshots(client *rancher.Client, localclusterID string, clusterName
 		return "", err
 	}
 	var snapshotToBeRestored string
+	s3Prefix := onDemandPrefix + clusterName
 
-	err = kwait.Poll(5*time.Second, 5*time.Minute, func() (done bool, err error) {
+	err = kwait.Poll(5*time.Second, defaults.FiveMinuteTimeout, func() (done bool, err error) {
 		snapshotList, err := GetSnapshots(client, localclusterID, clusterName)
 		if err != nil {
 			return false, err
 		}
+
 		if len(snapshotList) == 0 {
 			return false, fmt.Errorf("no snapshots found")
 		}
 
+		if strings.Contains(snapshotList[0], s3Prefix) {
```

Comment:
Just an explanation on why this is indexed from zero

Reply:
It's currently expected behavior that when you create a backup for S3 that it is in two storage locations: local and S3. Regardless of location, the `on-demand-` prefix will be in either location. This check specifically is for if you plan to use S3 as a backup location.

Reply:
Talked offline about this. Going to leave a comment to make this point clear why it is indexed this way.